tfcompile_study/experiments/conv_fc/test_run.py

```diff
@@ -0,0 +1,45 @@
+import tensorflow as tf
+import sys
+sys.path.insert(0, "../../.")
```

Comment:
you should be able to avoid this by using features of pytest to make top level functions available to the subdirectories

Reply:
yeah fixed it :)